engines/experiment/runner/runner.go

```diff
@@ -1,17 +1,26 @@
 package runner
 
 import (
-	"context"
+	"encoding/json"
 	"net/http"
 )
 
-// ExperimentRunner is the generic interface for generating experiment configs for a
-// given request
+type GetTreatmentOptions struct {
+	TuringRequestID string
```

Comment:
I don't think this is necessary for now, because the `router_version` is still expected to be captured in log messages. We are passing an instance of zap.Logger (the one that contains `router_version` extra key) to be used for producing log messages from the plugin: https://github.com/gojek/turing/blob/c97b8f09a43c4b74232d9708bd3c33dec38dad03/engines/experiment/plugin/rpc/rpc.go#L23. The way how it works is that host application (turing-router) reads stderr/stdout of the plugin process, parses log messages and then outputs it into stdout of the host application with provided logger.

Reply:
Thanks for the clarification - I hadn't paid attention to this.